model/riscv_insts_zicsr.sail

```diff
@@ -103,6 +103,9 @@ function readCSR csr : csreg -> xlenbits = {
     (0x31A, 32) => menvcfg.bits[63 .. 32],
     (0x320,  _) => zero_extend(mcountinhibit.bits),
 
+    /* Hardware Performance Monitoring event selection */
+    (0b0011001 /* 0x320 */ @ index : bits(5), _) if index >_u 0b00010 => read_mhpmevent(hpmidx_from_bits(index)),
```

Comment:
```suggestion
    (0b0011001 /* 0x320 */ @ index : bits(5), _) if unsigned(index) > 2 => read_mhpmevent(hpmidx_from_bits(index)),
```
I guess this would avoid the need for the new overloads? Not sure it's any easier to read though

Reply:
Done